src/components/shared/Paragraph.tsx

```diff
@@ -13,15 +13,26 @@ export default function Paragraph({
 }) {
   const CHARACTER_LIMIT_EXCEEDED =
     (characterLimit && description.length > characterLimit) ||
-    description.split('\n').length > 1
+    description.split('\n').length > 3
 
   const [expanded, setExpanded] = useState<boolean>(false)
   const toggleExpanded = () => setExpanded(!expanded)
 
-  // truncate the first line of the description
+  const allLines = description.split('\n')
+  let firstThreeLines = ''
+
+  // Truncate the first 3 lines of the description
+  allLines.forEach((line, index) => {
```

Comment:
you probably only need to loop over the first three lines with slice(0, 3), and then could join them with a new line:
```ts
const firstThreeLines = allLines
  .slice(0, 3)
  .join('\n')
```

wdyt?

Reply:
You could even possibly put all of this into the memo itself so it only runs when the character limit, or the description changes

Reply:
> you probably only need to loop over the first three lines with slice(0, 3), and then could join them with a new line:
> 
> ```ts
> const firstThreeLines = allLines
>   .slice(0, 3)
>   .join('\n')
> ```
> 
> wdyt?

Beautiful ser

Reply:
> ```ts
> const firstThreeLines = allLines
>   .slice(0, 3)
>   .join('\n')
> ```

Yep another great call